lib/api/liskApi.js

```diff
@@ -45,8 +45,8 @@ function LiskAPI (options) {
 
 	this.options = options;
 	this.ssl = options.ssl || false;
+	this.noRandomNode = options.noRandomNode || false;
 	this.testnet = options.testnet || false;
-	this.autoFindNode = options.autoFindNode || true;
```

Comment:
Totally agree! I just was not able to find such a good name :)

Thanks Oliver!